public/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <title>Project Atelier</title>
     <script defer src="bundle.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
I added this line in index.html as well for icons. Honestly, I'm not sure if this is most effective way in the long-run (I know that you could also install font-awesome using npm) but we can totally discuss this later!